app/views/subnets/import.html.erb

```diff
@@ -1,19 +1,19 @@
 <% title _("Import IPv4 subnets") %>
 
-<h2><%= _("The following IPv4 subnets have been found") %></h2>
```

Comment:
This was probably still useful, and could even be expanded to say to review the details in the accordion below.